src/Exception/InvalidMiddlewareException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace League\Tactician\Exception;
+
+class InvalidMiddlewareException extends \LogicException implements Exception
```

Comment:
Needs a little docblock and maybe `InvalidArgumentException` is a better base class here?